openapi/gateway.json

```diff
@@ -1197,24 +1197,12 @@
         "type": "object",
         "properties": {
           "control_plane": {
-            "type": "array",
-            "items": {
-              "type": "integer",
-              "format": "uint8",
-              "minimum": 0
-            },
-            "minItems": 32,
-            "maxItems": 32
+            "type": "string",
+            "format": "hex string (32 bytes)"
```

Comment:
Just making sure I understand - this field is effectively free form / up to clients to interpret, assuming it isn't one of the listed formats in the OpenAPI spec (a very short list)? Should it be something that's easier to parse by a machine? `"hex"` is appealing but loses the length requirement, for example.

Reply:
Yeah, it's a free-form field. I think we could teach progenitor about a format like "hex32" (to match e.g. uint64) and then switch to that in the future.

Reply:
(maybe "hex256")

Reply:
Gotcha, makes sense. Thanks.